pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -243,6 +243,26 @@ public static class Broker {
         Math.max(1, Math.min(10, Runtime.getRuntime().availableProcessors() / 2));
     // Same logic as CombineOperatorUtils
 
+    // Config for Jersey ThreadPoolExecutorProvider.
+    // By default, Jersey uses the default unbounded thread pool to process queries.
+    // By enabling it, BrokerManagedAsyncExecutorProvider will be used to create a bounded thread pool.
+    public static final String CONFIG_OF_ENABLE_BOUNDED_HTTP_ASYNC_EXECUTOR =
```

Comment:
I don't think it always associates with `http`. To make it specific, shall we name it `jersey.threadpool.executor`?

Reply:
Thanks Jackie, addressed in the third commit.